packages/rpc-rx/src/index.ts

```diff
@@ -130,10 +131,8 @@ export default class RpcRx implements RpcRxInterface {
     };
   }
 
-  private createSubject (name: string, params: Array<any>, section: RpcInterface$Section, unsubCallback?: () => void): ReplaySubject<any> {
-    const subject = new ReplaySubject(1);
-
-    Observable
+  private createSubject (name: string, params: Array<any>, section: RpcInterface$Section): ReplaySubject<any> {
```

Comment:
The previous `unsubCallback` is not actually used/passed anywhere (and this function is private), so removed it as well.